src/components/ConfigureEmail.tsx

```diff
@@ -55,22 +43,26 @@ export const ConfigureEmail = (props: ConfigureEmailProps) => {
           accessToken,
         ).then(() => {
           displayToast('Email has been successfully added', 'success')
+          history.replaceState(
```

Comment:
After we process the emailValidationSignedToken from the search params, this clears it out from the URL